src/ChurchCRM/data/Country.php

```diff
@@ -5,28 +5,35 @@ class Country implements \JsonSerializable
 {
   private string $countryCode;
   private string $countryName;
-  private ?string $countryNameYasumi = null;
-  public function __construct (string $CountryCode, string $CountryName, string $CountryNameYasumi = null)
+  private ?string $countryNameYasumi;
+
+  public function __construct(string $CountryCode, string $CountryName, string $CountryNameYasumi = null)
   {
     $this->countryCode = $CountryCode;
     $this->countryName = $CountryName;
     $this->countryNameYasumi = $CountryNameYasumi;
   }
 
-  public function getCountryCode(){
+  public function getCountryCode()
+  {
     return $this->countryCode;
   }
 
-  public function getCountryName() {
+  public function getCountryName()
```

Comment:
I realized I missed this

```suggestion
  public function getCountryName(): string
```